dockstore-webservice/src/main/java/io/dockstore/webservice/core/OrcidAuthor.java

```diff
@@ -2,41 +2,25 @@
 
 import io.swagger.annotations.ApiModel;
 import io.swagger.annotations.ApiModelProperty;
-import java.sql.Timestamp;
 import javax.persistence.Column;
 import javax.persistence.Entity;
-import javax.persistence.GeneratedValue;
-import javax.persistence.GenerationType;
-import javax.persistence.Id;
+import javax.persistence.NamedQueries;
+import javax.persistence.NamedQuery;
 import javax.persistence.Table;
-import org.hibernate.annotations.CreationTimestamp;
-import org.hibernate.annotations.UpdateTimestamp;
 
 @ApiModel(value = "OrcidAuthor", description = "This describes an ORCID-author of a version in Dockstore")
 @Entity
 @Table(name = "orcidauthor")
-public class OrcidAuthor {
-
-    @Id
-    @GeneratedValue(strategy = GenerationType.IDENTITY)
-    @ApiModelProperty(value = "Implementation specific ID for the author in this web service", required = true, position = 0)
-    private long id;
+@NamedQueries({
+        @NamedQuery(name = "io.dockstore.webservice.core.OrcidAuthor.findByOrcidId", query = "SELECT o FROM OrcidAuthor o WHERE o.orcid = :orcidId")
+})
+public class OrcidAuthor extends AbstractAuthor {
 
     @Column(columnDefinition = "varchar(50)", nullable = false)
-    @ApiModelProperty(value = "ORCID id of the author", required = true, position = 1)
+    @ApiModelProperty(value = "ORCID id of the author", required = true, position = 2)
```

Comment:
we're likely to query this table by orcid id (as you do above in the named query) 
so this could use an index, this will also enforce uniqueness which is a bonus